Bottom: 3010f75a01dc801a434efb51c2b7bb75bcff1b3c
Top:    6169782113d9db19b3ca564049805ec54c2478a1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 01:22:40 +0100

Refresh of crasy-test

---

diff --git a/src/draw_funcs.c b/src/draw_funcs.c
index 49beeba..8a2d557 100644
--- a/src/draw_funcs.c
+++ b/src/draw_funcs.c
@@ -45,15 +45,17 @@ draw_via_mask (PinType *via, const BoxType *drawn_area, void *userdata)
 static void
 draw_hole (PinType *pv, const BoxType *drawn_area, void *userdata)
 {
-  gui->set_line_cap (Output.fgGC, Round_Cap);
-  gui->set_line_width (Output.fgGC, 0);
-
   if (!TEST_FLAG (THINDRAWFLAG, PCB))
     gui->fill_circle (Output.bgGC, pv->X, pv->Y, pv->DrillingHole / 2);
 
   if (TEST_FLAG (THINDRAWFLAG, PCB) || TEST_FLAG (HOLEFLAG, pv))
-    gui->draw_arc (Output.fgGC, pv->X, pv->Y,
-                   pv->DrillingHole / 2, pv->DrillingHole / 2, 0, 360);
+    {
+      gui->set_line_cap (Output.fgGC, Round_Cap);
+      gui->set_line_width (Output.fgGC, 0);
+
+      gui->draw_arc (Output.fgGC, pv->X, pv->Y,
+                     pv->DrillingHole / 2, pv->DrillingHole / 2, 0, 360);
+    }
 }
 
 static void
